GameFiles/_integration.gsc

```diff
@@ -61,7 +76,7 @@ OnPlayerConnect()
         
         level.iw4adminIntegrationDebug = GetDvarInt( "sv_iw4madmin_integration_debug" );
         
-        if ( player.pers["isBot"] ) 
+        if ( isDefined(player.pers["isBot"]) && player.pers["isBot"] ) 
```

Comment:
`isDefined( player.pers["isBot"] )`

I forgot that gsc truthy logic is pretty dumb. "Oh it's not defined? that'll be true then"

Reply:
It is even worse on PlutoIW5 as the code execution stops right there for all human players and none of the threads needed for each player are spawned.